tests/unit/test_configurator/test_configurator_metrics_exporter.py

```diff
@@ -16,26 +16,6 @@
 
 
 class TestConfiguratorMetricsExporter:
-    def test_configure_metrics_exporter_disabled(
```

Comment:
Removed this and other similar test because this helper now assumes top `_configure` function does the enabled/disabled check.